lib-injection/README.md

```diff
@@ -1,4 +1,12 @@
-# Host injection
+### Injection
+
+The inject script would add `ddtrace` to your application's `Gemfile` to [instrument your application](https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby/#rails-or-hanami-applications)).
+
+* Supports `Ruby on Rails` / `Hanami` application
+* Supports `Bundler` version must be >= 2.3
+* Does not support vendoring gem ([Bundler's Deployment Mode](https://www.bundler.cn/man/bundle-install.1.html#DEPLOYMENT-MODE) or setting `BUNDLE_PATH`)
```

Comment:
Is there an example of what vendoring gem means? (Ie: It's not clear to me that this would affect the frozen gemfile scenario).

Reply:
Aren't these two different things?
* Vendoring with BUNDLE_PATH.
* Deployment mode.

If they are different, we should make it clear that these are not both the same kind of "vendoring gem".